### ISSUE TITLE
change learning rate

### ISSUE THREAD

**Issue by avacaondata** [sueiras/deep_learning_images#1]
Una posibilidad es usar esto: def scheduler(epoch):
    if epoch == 5:
        model.lr.set_value(.02)
    return model.lr.get_value()

change_lr = LearningRateScheduler(scheduler)

model.fit(x_embed, y, nb_epoch=1, batch_size = batch_size, show_accuracy=True,
       callbacks=[chage_lr])

Pero te adjunto mi práctica de perros y gatos, ahí tengo una implementación que es algo mejor porque cambia solo el lr si aquello está dejando de mejorar:

[alejandro_vaca_deep_learning_def.zip](https://github.com/sueiras/deep_learning_images/files/3132533/alejandro_vaca_deep_learning_def.zip)

**Comment by sueiras** [comment]
Muchas gracias Alejandro. Efectivamente esto ofrece bastantes posibilidades.
